### ISSUE TITLE
Mathematics

### ISSUE THREAD

**Issue by jinmang2** [jinmang2/bring_it_on#7]
- [Matrix Calculus](https://ccrma.stanford.edu/~dattorro/matrixcalc.pdf)
- [The Matrix Cookbook, UCI](https://www.ics.uci.edu/~welling/teaching/KernelsICS273B/MatrixCookBook.pdf)

**Comment by jinmang2** [comment]
https://math.stackexchange.com/questions/1733814/derivative-of-elementwise-function-working-on-a-vector

https://web.stanford.edu/class/cs224n/readings/gradient-notes.pdf

https://math.stackexchange.com/questions/734649/derivative-of-a-matrix-outer-product-chain-rule

https://mmuratarat.github.io/2019-02-10/derivative-of-softmax-loss

https://darkpgmr.tistory.com/141

https://en.wikipedia.org/wiki/Spectral_theorem

https://math.stackexchange.com/questions/1064016/linear-combination-of-eigenvectors-corresponding-to-the-same-eigenvalue

https://onlinelibrary.wiley.com/doi/pdf/10.1002/0471705195.app3

https://www.cs.huji.ac.il/~csip/tirgul3_derivatives.pdf

https://math.stackexchange.com/questions/1461820/linear-combination-of-gradient-vector-fields

https://www.cs.cmu.edu/~zkolter/course/15-884/linalg-review.pdf

http://www.ams.sunysb.edu/~jiao/teaching/ams526_fall18/lectures/lecture21.pdf

https://math.stackexchange.com/questions/1652414/l2-matrix-norm-upper-bound-in-terms-of-bounds-of-its-column

https://math.stackexchange.com/questions/1019094/prove-that-the-largest-singular-value-of-a-matrix-is-greater-than-the-largest-ei

https://math.stackexchange.com/questions/586663/why-does-the-spectral-norm-equal-the-largest-singular-value/586835

https://math.stackexchange.com/questions/2583417/spectral-radius-is-not-matrix-norm

- 얘는 중요!! Lemma 10, Lemma 11 Spectral radius
https://www.math.drexel.edu/~foucart/TeachingFiles/F12/M504Lect6.pdf

https://math.stackexchange.com/questions/127500/what-is-the-difference-between-singular-value-and-eigenvalue

**Comment by jinmang2** [comment]
https://jeongchul.tistory.com/603

https://en.wikipedia.org/wiki/Matrix_calculus

https://bskyvision.com/171

https://math.stackexchange.com/questions/586663/why-does-the-spectral-norm-equal-the-largest-singular-value/586835

https://math.stackexchange.com/questions/1019094/prove-that-the-largest-singular-value-of-a-matrix-is-greater-than-the-largest-ei

https://www.math.drexel.edu/~foucart/TeachingFiles/F12/M504Lect6.pdf

**Comment by jinmang2** [comment]
# Gershgorin circle theorem

https://en.wikipedia.org/wiki/Gershgorin_circle_theorem

https://mathworld.wolfram.com/GershgorinCircleTheorem.html

https://www.youtube.com/watch?v=tCvRJA7T7qg

https://www.youtube.com/watch?v=Cd2Nty4V-dk

https://math.la.asu.edu/~gardner/Gcircle.pdf

http://math.stmarys-ca.edu/wp-content/uploads/2017/07/David-Marquis.pdf

https://www.math.tamu.edu/~wrobel/GIGEM2018/Talks/handout_eifler_GIG'EM.pdf

http://buzzard.ups.edu/courses/2007spring/projects/brakkenthal-paper.pdf

https://www.math.uwaterloo.ca/~zcramer/Gershgorin%20GSC.pdf

https://www.geogebra.org/m/wDEj3Xg9

https://gall.dcinside.com/mgallery/board/view/?id=math&no=20468&_rk=urT&page=1

https://www.wikizero.com/ko/토론:게르시고린의_정리

https://wordrow.kr/의미/게르시고린의%20정리/

http://www.mathnet.ru/links/b5ddfed311969f2218b48652add349d1/im5235.pdf

https://www.mpp.mpg.de/~jingliu/ECPI/PowerMethodProof.pdf

https://27x37.files.wordpress.com/2011/05/mcgraw-hill-numerical-methods-using-matlab.pdf

**Comment by jinmang2** [comment]
Softmax vs Logsoftmax

https://papers.nips.cc/paper/2018/file/9dcb88e0137649590b755372b040afad-Paper.pdf
https://stackoverflow.com/questions/61567597/how-is-log-softmax-implemented-to-compute-its-value-and-gradient-with-better
https://discuss.pytorch.org/t/logsoftmax-vs-softmax/21386/9
https://stackoverflow.com/questions/44081007/logsoftmax-stability
https://papers.nips.cc/paper/2018/file/9dcb88e0137649590b755372b040afad-Paper.pdf
https://royalsocietypublishing.org/doi/10.1098/rsif.2015.0330
https://blog.metaflow.fr/ml-notes-why-the-log-likelihood-24f7b6c40f83
https://math.stackexchange.com/questions/892832/why-we-consider-log-likelihood-instead-of-likelihood-in-gaussian-distribution
https://zhang-yang.medium.com/understanding-cross-entropy-implementation-in-pytorch-softmax-log-softmax-nll-cross-entropy-416a2b200e34
https://discuss.pytorch.org/t/logsoftmax-vs-softmax/21386/15

**Comment by jinmang2** [comment]
Kenji Kawaguchi
- ResNet에 대해서만 증명
- ㄻㄴㅇ
https://proceedings.neurips.cc/paper/2016/file/f2fc990265c712c49d51a18a32b39f0c-Paper.pdf
